content/src/content/jcr_root/apps/core/wcm/components/progressbar/v1/progressbar/progressbar.html

```diff
@@ -17,9 +17,9 @@
      id="${progressBar.id}"
      class="cmp-progressbar"
      data-cmp-data-layer="${progressBar.data.json}">
-    <span class="cmp-progressbar__label--completed">${'#.##' @ format=progressBar.completed}</span>
-    <span class="cmp-progressbar__label--remaining">${'#.##' @ format=progressBar.remaining}</span>
-    <div class="cmp-progressbar__bar" role="progressbar"
+    <span class="cmp-progressbar__label--completed" aria-hidden="true">${'#.##' @ format=progressBar.completed}</span>
+    <span class="cmp-progressbar__label--remaining" aria-hidden="true">${'#.##' @ format=progressBar.remaining}</span>
+    <div class="cmp-progressbar__bar" role="progressbar" aria-label="${'#.##' @ format=progressBar.completed}% completed"
```

Comment:
We should also add `i18n` to the HTL expression

Reply:
> We should also add `i18n` to the HTL expression

Added in 7a6a38cc0fece56c59b5e5ada9e375ce71de90f4.